guides/release/using-typescript/core-concepts/routing.md

```diff
@@ -44,10 +82,9 @@ How that works:
 
 - `Resolved<P>` says "if this is a promise, the type here is whatever the promise resolves to; otherwise, it's just the value"
```

Comment:
We should update this to use the `Awaited<T>` type that TS ships natively as of late in the 4.X series!